lib/smart_app_launch/client_suite/access_bsca_interaction_test.rb

```diff
@@ -13,6 +13,13 @@ class SMARTClientAccessBackendServicesConfidentialAsymmetricInteraction < Infern
       During this test, Inferno will wait for the client to access data
       using a SMART token obtained using the Backend Services flow.
     )
+    verifies_requirements 'hl7.fhir.uv.smart-app-launch_2.2.0@229',
+                          'hl7.fhir.uv.smart-app-launch_2.2.0@233',
+                          'hl7.fhir.uv.smart-app-launch_2.2.0@235',
+                          'hl7.fhir.uv.smart-app-launch_2.2.0@236',
+                          'hl7.fhir.uv.smart-app-launch_2.2.0@237',
```

Comment:
@237 seems a better fit for 5.11 (already there). Remove from here.

@237
[when requesting] an access token via HTTP POST to the FHIR authorization server’s token endpoint URL [the] `client_assertion_type` [parameter is] `required` [and SHALL contain] … Fixed value: `urn:ietf:params:oauth:client-assertion-type:jwt-bearer`

Reply:
Removed in [9b4d4ad](https://github.com/inferno-framework/smart-app-launch-test-kit/pull/105/commits/9b4d4adf0a9945b994d01c6ee21808ba05be91a6).